### ISSUE TITLE
Please contact us about your verificatum repository. It constitutes a copyright infringement.

### ISSUE THREAD

**Issue by verificatum** [edufer/dl4java-examples#1]
Dear Eduardo,

You keep our copyrighted code at GitHub, namely verificatum-full-1.0.8 (your "verificatum"-repository). 

To avoid that we have to contact GitHub directly and take legal action, I ask you to email me to initiate a correspondence. I have already tried to email you at e.fernandes@appgree.com, but you have not answered, so I try to contact you like this.

Best regards,

Douglas Wikström
Verificatum AB
douglas@verificatum.com
+46 73 687 0060

**Comment by edufer** [comment]
Hi.

Sorry about that. I was not aware about the copyright when I make the
github fork.

I just saw a message in my github message inbox and I proceeded immediately
to delete the repository. Thanks for warning me about that.

I apologize for any inconvenience I could generating sharing it.

Thanks for your email.

Regards and let me congratulate you for the contribution you're doing in
electronic voting.

/Eduardo
